README.md

```diff
@@ -1,2 +1,242 @@
-# eeajs
-EEA JavaScript libraries.
+# EEA JavaScript libraries - EEAJS
+EEA JavaScript libraries to interact with Pantheon-Quickstart exploring the privacy feature nd deploy contracts.
+
+## Getting Started
+
+To start clone the repository:
+
+```bash
+$ git clone https://github.com/PegaSysEng/eeajs.git
+```
+
+## Prerequisites
+
+To run this tutorial, you must have the following installed:
+
+- [Docker and Docker-compose](https://docs.docker.com/compose/install/) 
+
+- [Git command line](https://git-scm.com/)
+
+- MacOS or Linux 
+    
+    !!! important 
+        The Private Network Quickstart is not supported on Windows. If using Windows, run the quickstart
+        inside a Linux VM such as Ubuntu. 
+
+- [Pantheon-Quickstart](https://github.com/PegaSysEng/pantheon-quickstart)
+    - [Privacy](https://github.com/PegaSysEng/pantheon-quickstart/tree/4-node-example/privacy)
+
+- [Nodejs](https://nodejs.org/en/download/)
+
+## Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+Access the folder and install the dependencies.
+
+```bash
+$ cd eeajs/
+$ npm install
+```
+
+Execute `node example/eventEmitter.js`.
```

Comment:
I'd add a heading before this - it's not really installing. Deploy Contracts or something like that.